packages/contracts-bedrock/scripts/DeploySuperchain.s.sol

```diff
@@ -258,25 +264,16 @@ contract DeploySuperchain is Script {
         dso.writeOutputFile(_outfile);
     }
 
-    // This entrypoint is for use with Solidity tests, where the input and outputs are structs.
-    function run(DeploySuperchainInput.Input memory _input) public returns (DeploySuperchainOutput.Output memory) {
-        // Solidity without file IO, so etch the IO helper contracts.
-        (DeploySuperchainInput dsi, DeploySuperchainOutput dso) = etchIOContracts();
-
-        // Load the input struct into the input contract.
-        dsi.loadInput(_input);
-
-        // Run the deployment script and write outputs to the DeploySuperchainOutput contract.
-        run(dsi, dso);
-
-        // Return the output struct from the output contract.
-        return dso.output();
-    }
```

Comment:
comment only: Removing this looks like a bit of an ergonomic setback based on the diff in the test files.

Reply:
I think it's less ergonomic in these test files because we need to use the individual getters, but more ergonomic at scale (when we have feature-versions that inherit from these bases with different interfaces) for that same reason.